.github/workflows/dash.yaml

```diff
@@ -25,6 +25,8 @@ on:
   pull_request:
     branches:
       - main
+    paths:
+      - "package-lock.json"
```

Comment:
we should include the package.json as well
```suggestion
      - package-lock.json
      - package.json
```

Reply:
thx for the hint